specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerAppsJob_Run.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppsJobName": "testcontainerAppsJob0",
+    "api-version": "2022-11-01-preview"
+  },
+  "responses": {
+    "202": {}
```

Comment:
Is there no operation id to get status of job run

Reply:
@jijohn14 currently there is no operation Id associated with a single run. We had discussions around it.
Adding @vinisoto @trajkobal